src/js/models/search/SearchOperation.js

```diff
@@ -152,6 +161,21 @@ class SearchOperation {
             }
         }
 
+        // Add Pricing Type Queries
+        if (this.pricingType.length > 0) {
```

Comment:
Adding searchContext to support the above issue w/ filters not hitting correct fields on various endpoints.